### PR TITLE
feat: unify connection events

### PR DESCRIPTION
**Pull request by tomiir**
# Breaking Changes

N/A

# Changes

- feat:
- fix: add name to connect event. Remove external type and replace with browser type
- chore:

# Associated Issues

closes 
[#1928](https://github.com/WalletConnect/web3modal/issues/1928)

### CODE REVIEW

File: packages/core/tests/controllers/AssetController.test.ts [hunk 1]
```diff
@@ -8,7 +8,8 @@ describe('AssetController', () => {
       walletImages: {},
       networkImages: {},
       connectorImages: {},
-      tokenImages: {}
+      tokenImages: {},
+      currencyImages: {}
```
Comment (ignaciosantise) [inline packages/core/tests/controllers/AssetController.test.ts:12]: is this change intended?

File: packages/core/tests/controllers/TransactionsController.test.ts [hunk 2]
```diff
@@ -9,7 +9,8 @@ describe('TransactionsController', () => {
       transactionsByYear: {},
       loading: false,
       empty: false,
-      next: undefined
+      next: undefined,
+      coinbaseTransactions: []
```
Comment (ignaciosantise) [inline packages/core/tests/controllers/TransactionsController.test.ts:13]: same

File: packages/core/tests/controllers/AssetController.test.ts (hunk 1 above)
Comment (tomiir) [reply packages/core/tests/controllers/AssetController.test.ts:12]: yes

**Review by dnul — APPROVED** [review]
lgtm, see comments

File: packages/scaffold/src/views/w3m-connecting-external-view/index.ts [hunk 3]
```diff
@@ -54,7 +54,7 @@ export class W3mConnectingExternalView extends W3mConnectingWidget {
         EventsController.sendEvent({
           type: 'track',
           event: 'CONNECT_SUCCESS',
-          properties: { method: 'external' }
+          properties: { method: 'browser', name: this.connector.name || 'Unknown' }
```
[the remark this answers is not in the document]
Comment (tomiir) [reply packages/scaffold/src/views/w3m-connecting-external-view/index.ts:53]: yessir

**Review by svenvoskamp — APPROVED** [review]
